docs/whatsnew/5.3.rst

```diff
@@ -37,6 +37,42 @@ respectively.
     >>> cosmo.comoving_distance(0.5)  # doctest: +SKIP
     <Quantity 1982.66012926 Mpc>
 
+.. _whatsnew-5.3-compressed-fits:
+
+Efficient data access for compressed FITS files
+===============================================
+
+In previous astropy versions, when accessing data for a compressed
+FITS file via the :attr:`~astropy.io.fits.CompImageHDU.data` property of
+:class:`~astropy.io.fits.CompImageHDU`,
+the whole data was read in and decompressed even if a small part of
+the data was required. The :class:`~astropy.io.fits.CompImageHDU` class
+now has a :attr:`~astropy.io.fits.CompImageHDU.section` property which
+can be used to access specific sections of the data and will result in
+as little data as possible being read in and decompressed. The following
+example shows a speedup of 60x when accessing a 8192 by 8192 image
+with 256 by 256 tiles but larger speedups are possible for larger data
+and/or smaller tiles:
+
+**Before**::
```

Comment:
I don't think this is very useful to add, since the reader cannot try this themselves.

Reply:
Would it be more useful to add code to generate the file or remove the example altogether?

Reply:
I would just remove the example. Though I must add that I missed that it was in the what's new, not the documentation proper. So, less of an issue to have an example that cannot be checked by the user. Still, overall, I think just remove it.

Reply:
Done, though I've still kept a minimal example to just show *how* to use ``section`` which I think is important.